salvo/src/lib/api/image.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package salvo;
+
+// Capture all docker images required for the benchmark. This object is examined
+// first before evaluating the state of specified source locations
+message DockerImages {
+  // Determines whether required docker are used if they already exist. The
+  // benchmark image and binary image must be specified. If this is set to false
+  // there must be a specified source location from which we build the image
+
+  // This should be implicit. Images are preferred over building from source. We
```

Comment:
I don't really understand the comment as it stands. "This should be implicit." sounds like a wish and I am not sure in what context we are making the wish and why it isn't under our control.

Maybe we can improve the comment by instead focusing on describing the meaning of the proto field?